projects/postgresql/build.sh

```diff
@@ -24,7 +24,12 @@ CC="" CXX="" CFLAGS="" CXXFLAGS="" su fuzzuser -c ../configure
 cd src/backend/fuzzer
 su fuzzuser -c "make createdb"
 chown -R root .
-cp -r temp $OUT/
+cp -r temp/ query_db
```

Comment:
better just have one tar.gz, no need to have two with just the prefix path different.